functests/install-script.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+
+# This file is part of MinIO DirectPV
+# Copyright (c) 2022 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+echo "Installing direct PV"
+
+operating_system=""
+darwin="Darwin"
+linux="Linux"
+windows="Windows"
+
+function directpv(){
+    read operating_system kubectl_url minikube_url < <(initialize)
+    verify_kubectl
+    verify_krew 
+    kubectl krew install directpv
+    kubectl directpv install
+}
+
+# Fetch the operating system
+function initialize(){
+    case "$OSTYPE" in
+        darwin*)  
+            operating_system=$darwin            
+            kubectl_url="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
+            minikube_url="https://storage.googleapis.com/minikube/releases/latest/minikube-darwin-amd64"
+            ;;
+        linux*)   
+            operating_system=$linux
+            kubectl_url="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+            minikube_url="https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64"
+            ;;
+        msys*)    
+            operating_system=$windows 
+            kubectl_url="https://dl.k8s.io/release/v1.23.0/bin/windows/amd64/kubectl.exe";;
+        *)        echo "unknown: $OSTYPE" ;;
+    esac
+    echo "$operating_system $kubectl_url $minikube_url"
+}
+
+
+function verify_kubectl() {
+    # Install kubectl if not present
+    if ! [ "$(kubectl version --client)" ]; then
+        install_kubectl
+    fi
+    # Install minikube and configure kubectl to use it
+    if ! [ "$(kubectl version)" ]; then
+        verify_minikube
+    fi
+}
+
+function install_kubectl() {
+    case "$operating_system" in 
+        darwin)
+            cd /usr/local/bin/ 
+            curl -LO $kubectl_url 
+            chmod +x kubectl 
+            cd - ;;
+        linux)
+            cd ~/.local/bin
+            curl -LO $kubectl_url 
+            chmod +x kubectl 
+            cd - ;;
+        windows)
+            curl -LO $kubectl_url
+            PATH=%PATH%;$PWD ;;
```

Comment:
`Warning: ./functests/install-script.sh:81:13: warning: PATH is the shell search path. Use another name. [SC2123]`

You cannot mix MS Windows PATH variable and bash PATH variable notation.

Reply:
This PR caters Linux only .